medusa/scene_numbering.py

```diff
@@ -196,32 +221,33 @@ def find_xem_absolute_numbering(series_obj, absolute_number):
         return int(rows[0]['scene_absolute_number'])
 
 
-def get_scene_numbering(series_obj, episode, season=None):
+def get_scene_numbering(series_obj, scene_episode, scene_season=None):
```

Comment:
This is all very confusing to me.

Shouldn't this be `get_indexer_numbering_from_scene`?

Reply:
Yeah. That would be an improvement. I agree